### ISSUE TITLE
AttributeError: module 'eeweather' has no attribute 'match_zcta'

### ISSUE THREAD

**Issue by jakesherman** [openeemeter/eeweather#16]
Good morning,

I tried using the `match_zcta` function in `eeweather` (I'm using Python 3.6.5):

```python
import eeweather
print(eeweather.__version__)
zcta = '70001'
result = eeweather.match_zcta('70001')
result
```

Which resulted in:

```
0.2.1
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-857f81446776> in <module>()
      2 print(eeweather.__version__)
      3 zcta = '70001'
----> 4 result = eeweather.match_zcta('70001')
      5 result

AttributeError: module 'eeweather' has no attribute 'match_zcta'
```

It looks like the `match_zcta` function, while referenced in the documentation, doesn't actually exist in the package/the master branch. I'm wondering if `match_zcta` exists in another branch, and maybe hasn't been merged into master yet?

**Comment by jakesherman** [comment]
@philngo Awesome, that's exactly the workaround that I was doing! Thank you.

```
zcta = '70001'
result = eeweather.match_lat_long(*eeweather.zcta_to_lat_long(zcta))
result
```

**Comment by philngo** [comment]
Released version `0.2.2` which updates the docs and adds `requests`, `pyproj`, and `shapely` to the required packages.
